.github/workflows/integ-test.yml

```diff
@@ -208,6 +209,7 @@ jobs:
       run:
         working-directory: ${{ env.WORKDIR }}
     strategy:
+      max-parallel: 1
```

Comment:
I think this would not result in each sc_host running at most 1 test. We provide a matrix with "sc_host, test_name" fields. To github sc_host is not more important than test_name.

But something like this might work.

```
concurrency:
  group: ci-${{ matrix.sc_host }}
```

Reply:
Good call. I thought the order you declare the matrix values defined the execution order.